api/tests/unit/features/test_unit_features_permissions.py

```diff
@@ -20,374 +21,453 @@
 )
 from users.models import FFAdminUser, UserPermissionGroup
 
-mock_view = mock.MagicMock()
-mock_request = mock.MagicMock()
 
-feature_permissions = FeaturePermissions()
+def test_organisation_admin_can_list_features(
+    admin_user: FFAdminUser,
+    project: Project,
+) -> None:
+    # Given
+    feature_permissions = FeaturePermissions()
+    mock_view = mock.MagicMock(
+        kwargs={"project_pk": project.id},
+        detail=False,
+        action="list",
+    )
+    mock_request = mock.MagicMock(data={}, user=admin_user)
 
+    # When
+    result = feature_permissions.has_permission(mock_request, mock_view)
 
-@pytest.mark.django_db
-class FeaturePermissionsTestCase(TestCase):
-    def setUp(self) -> None:
-        self.organisation = Organisation.objects.create(name="Test")
-        self.project = Project.objects.create(
-            name="Test project", organisation=self.organisation
-        )
+    # Then
+    assert result is True
 
-        self.feature = Feature.objects.create(name="Test feature", project=self.project)
 
-        self.user = FFAdminUser.objects.create(email="user@test.com")
-        self.user.add_organisation(self.organisation, OrganisationRole.USER)
+def test_project_admin_can_list_features(
+    admin_user: FFAdminUser,
```

Comment:
Hmm... this isn't correct. It should be `staff_user` (and we should give them project admin permissions). Something like: 

```
UserProjectPermission.objects.create(user=staff_user, admin=True)
```